### DISCUSSION TITLE
Possible problem with the saving of pm5s files (Layer Exposure is set to 0s)

### DISCUSSION THREAD

**Discussion by Flashhawk2K** [sn4k3/UVtools#757] [Q&A] [answered]
Hy there,
i posted also to the FB group, so maybe ignore this..
My AnyCubic M5s sets the layer exposure time from 4,25s to 0s, when i edited the slicer file befor in UVTools (which shows no signs of this). Can somebody tell me if this preventable?
Thank you in advance
FH
![377260275_3506824382910884_2162348266160794648_n](https://github.com/sn4k3/UVtools/assets/71667687/a4e49015-d212-4953-8638-b2090d1e7f5e)

**Comment by Flashhawk2K** [comment]
Here is a zip file with one unedited and one edited version. If the 0s problem exists in the unedited one i can not say, since i did not print it due to the resin traps...
https://we.tl/t-bVfrceDF9w

**Reply by Flashhawk2K** [reply]
Ok I tried to slice something else and edit it like the files above. And as it seems, it's working. So maybe it's the STL files? Should I send you the STL and lys files too?

**Reply by sn4k3** [reply, marked as answer]
Both files look correct, but one has per layer settings on which can causing trouble #742

**Reply by Flashhawk2K** [reply]
Great, thank you very much! I looked at the other files that I had issues with, and at least one has not set the perlayer flag. I reset the layer setting nonetheless and will try to print it. Maybe it works anyway. Will report back! 
Thank you again, for the help and the tool!

**Comment by Flashhawk2K** [comment]
Ok and it failed again with the first layer still in the vat... was not the file I send you, but had failed before and showed the layer issue you mentioned, which I corrected with  your workaround. Seems it did not help for my problem(s)... sorry to bother you again! Should I upload this file too?

**Reply by sn4k3** [reply]
Try to print the file without modify with UVtools

**Reply by Flashhawk2K** [reply]
Ok, it failed also. It's not only or even at all a problem of UVTools. But on the other hand, I successfully printed a file just the other day with the same settings!?! It was an stl that I pulled presupported of thingyverse but had a .lys file included were i edited the supports a bit.  Idk anymore what to do at this point...

**Reply by sn4k3** [reply]
If print is failing stuck on fep it may be settings or poor surface quality. If machine itself is the problem, try to update firmware first.

**Reply by Flashhawk2K** [reply]
Ok. so i think i found a solution to my peticular problem. I used very large files in hope of getting fine details, but seems that messed lychee up in some way. If i reduce the files with 3d Builder by 1 level, use UVTool to my full advantage, and reset the layer thingy: i can print... at least one model. I just started the big one with 2 models +2 bases and will see in 11h whats behind my theorie. Thank you again for looking into it!!

**Reply by Flashhawk2K** [reply]
Ok it works, it seems i just need to work on my supporting skills ;)
I will write into Lychee Support about the big file "issue" maybe they have a solution for that too...
